### ISSUE TITLE
ScenarioManager - add inputs and outputs arguments in constructor

### ISSUE THREAD

**Issue by victorterpstra** [IBM/dse-decision-optimization-utilities#26]
Add `inputs`and `outputs` arguments in ScenarioManager.__init__

Currently, you need to:
```excel_file_name = 'TestWriteExcel'
sm = ScenarioManager()
sm.inputs=inputs
sm.outputs=outputs
sm.write_data_to_excel(excel_file_name)
```

Would be cleaner if we can do:
```
excel_file_name = 'TestWriteExcel'
sm = ScenarioManager(inputs=inputs, outputs=outputs)
sm.write_data_to_excel(excel_file_name)
```

**Comment by victorterpstra** [comment]
Added to v0.5.1.1b

**Comment by victorterpstra** [comment]
v0.5.1.1b released as v0.5.2.0
